### PR TITLE
removing all tabs and trailing space

### PR DESCRIPTION
**Pull request by bam241**
just some cleaning to remove mix indentation....

### CODE REVIEW

**Comment by gonuke** [comment]
This needs a news file

**Comment by bam241** [comment]
also removed tab from python files (replacing them with 4spaces)

File: src/measure.cpp [hunk 1]
```diff
@@ -199,4 +199,5 @@ double measure( moab::EntityType type,
       return 0.0;
   }
 }
-      
+
+
```
Comment (gonuke) [inline src/measure.cpp]: Why this extra line?

**Review by gonuke — COMMENTED** [review]
Looks like you added a line (by accident?)

File: src/measure.cpp (hunk 1 above)
Comment (bam241) [reply src/measure.cpp]: an accident.

nice catch tho !

Comment (gonuke) [reply src/measure.cpp]: It was lucky...  When I review these whitespace PR's, I set Github to ignore whitespace and then I should see no other changes.  Although I would consider this to also be whitespace, it showed it anyway

**Comment by gonuke** [comment]
I think you need to restart tests...

**Comment by bam241** [comment]
We need to do something about those test... endf website seems not be reliable enough...
They might kick CI out because of the amount of queries it does...

**Comment by gonuke** [comment]
Thanks @bam241

**Pull request merged**